frontend/src/metabase/api/user.ts

```diff
@@ -85,6 +85,9 @@ export const userApi = Api.injectEndpoints({
       invalidatesTags: (_, error, { id }) =>
         invalidateTags(error, [listTag("user"), idTag("user", id)]),
     }),
+    listUserAttributes: builder.query<string[], void>({
+      query: () => "/api/mt/user/attributes",
```

Comment:
why `api/mt` and not `api/ee`? 😅

Reply:
This is an old endpoint, no clue 😅 I'm just adding it to our rtkquery definitions.